frontend/src/composables/useNormHtml.ts

```diff
@@ -28,3 +36,34 @@ export function useNormHtml(
 
   return readonly(normHtml)
 }
+
+export function useNormHtmlByEli(
+  eli: MaybeRefOrGetter<string | undefined>,
+  at?: MaybeRefOrGetter<Date | undefined>,
+): {
+  normHtml: Readonly<Ref<string | undefined>>
+  error: Readonly<Ref<Error | null>>
+  isFetching: Readonly<Ref<boolean>>
+} {
+  const eliRef = ref(toValue(eli))
+  const atRef = ref(toValue(at))
+
+  watch(
+    () => [toValue(eli), toValue(at)],
+    () => {
+      eliRef.value = toValue(eli)
+      atRef.value = toValue(at)
+    },
+    { immediate: true },
+  )
+
+  const { data, error, isFetching } = useGetNormHtmlByEli(eliRef, false, atRef)
+
+  const normHtml = computed(() => data.value ?? undefined)
+
+  return {
```

Comment:
thinking of it: is there any reason to return just these fields and not the whole thing `useGetNormHtmlByEli` returns?

And if no is there any reason to even keep this composable at all, its just calling `useGetNormHtmlByEli` at that point.